addons/message_update_v2.18/manifest.json

```diff
@@ -27,7 +27,7 @@
         "content": [
           {
             "id": "l_2",
-            "content": "A new “My Devices” screen that makes it easier to view and manage your devices"
+            "content": "The ability to upgrade to an annual plan from within the app"
```

Comment:
This string should be localized using machine memory as it was https://github.com/mozilla-l10n/mozilla-vpn-client-l10n/pull/391